content/cv/conv_and_padding.md

```diff
@@ -0,0 +1,218 @@
+---
+title: "Convolution & Padding"
+date: 2019-10-17T13:34:08+07:00
```

Comment:
Anh gợi ý đặt tên tiêu đề là "Toán tử Convolution", vì padding chỉ là 1 giải pháp kĩ thuật cho convolution, còn convolution mới thật sự là 1 phép toán.